### ISSUE TITLE
【SF】【Table】【A11y】More button lacks descriptive accessible name

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents#9807]
### Is your feature request related to a problem?

Recently, I've received an a11y issue about the accessible name for more button in `Table`. It will read "More Button" and we can't set the descriptive accessible name for it.

However, the A11y expert expect the similar behavior here. 
![image](https://github.com/user-attachments/assets/a6402abe-087c-4e48-ab17-6fdb5c369153)

### Describe the solution you'd like

Support setting accessible-name for more button in Table.

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by tandan0813** [comment]
Hi, the version of `@ui5/webcomponents` we use:
"@ui5/webcomponents": "1.24.0"
"@ui5/webcomponents-react": "1.28.4"

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-core ,
Can you please take a look at this issue/feature request for setting accessible-name for more button in Table.
Best Regards,
Konstantin

**Comment by ksblklu02** [comment]
Hi @simlin 
Now growing button text/subtext can be correctly applied as button aria-label, actually we want set an invisible accessibleName for the growing button. Do we have any solution to set invisible accessible name to the growing button?

**Comment by simlin** [comment]
Hi @ksblklu02 
no, there is no option to set an alternative label or accessible name for the growing button.
Can you please share the background for this? The growing button with text/subtext exist in a similar way also in the classical UI5 table/list and it follows the available specifications. What's the reason for an application to provide a special behavior which differs from other applications?
Thx and Best Regards
Jens
